instance creation
with: anException

	^self new 
		exception: anException